src/main/sensors/gyro.c

```diff
@@ -742,15 +746,38 @@ static bool isDynamicFilterActive(void)
     return feature(FEATURE_DYNAMIC_FILTER);
 }
 
-static void gyroInitFilterDynamicNotch(gyroSensor_t *gyroSensor)
+static void gyroInitFilterDynamic(gyroSensor_t *gyroSensor)
 {
-    gyroSensor->notchFilterDynApplyFn = nullFilterApply;
+    gyroSensor->dynFilterApplyFn = nullFilterApply;
 
     if (isDynamicFilterActive()) {
-        gyroSensor->notchFilterDynApplyFn = (filterApplyFnPtr)biquadFilterApplyDF1; // must be this function, not DF2
-        const float notchQ = filterGetNotchQ(400, 390); //just any init value
-        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-            biquadFilterInit(&gyroSensor->notchFilterDyn[axis], 400, gyro.targetLooptime, notchQ, FILTER_NOTCH);
+        switch (gyroConfig()->dyn_filter_type) {
+        case FILTER_PT1: {
+            const int dynamicLowpassDefaultCutoffHz = 200;
```

Comment:
Can we use defines for these constants to be more in line with the rest of the codebase please?

Reply:
Done